shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -59,7 +59,8 @@
 public class ScanCommand extends Command {
 
   private Option scanOptAuths, scanOptRow, scanOptColumns, disablePaginationOpt, showFewOpt,
-      formatterOpt, interpreterOpt, formatterInterpeterOpt, outputFileOpt;
+      formatterOpt, interpreterOpt, formatterInterpeterOpt, outputFileOpt, scanOptCfOptions,
+      scanOptColQualifier;
```

Comment:
```suggestion
      formatterOpt, interpreterOpt, formatterInterpeterOpt, outputFileOpt, scanOptColFam,
      scanOptColQualifier;
```

Doesn't have to be this specific wording but the two added opts should have a similar naming convention.